qutebrowser/config/configdata.yml

```diff
@@ -746,6 +746,17 @@ content.xss_auditing:
     Suspicious scripts will be blocked and reported in the inspector's
     JavaScript console.
 
+content.mute:
+  default: false
+  type: Bool
+  supports_pattern: true
+  desc: >-
+    Mute new web pages.
```

Comment:
"new web pages" isn't really clear - what about "Automatically mute tabs" or something? Maybe @mschilli87 has a better idea?

Reply:
Good point. I'll give it some sleep-thought until tomorrow. :wink:

Reply:
No better idea by now.So LGTM from me for the phrasing... :wink: